cmd/exchange_wrapper_issues/main.go

```diff
@@ -725,6 +726,37 @@ func testWrappers(e exchange.IBotExchange, base *exchange.Base, config *Config)
 			Error:      msg,
 			Response:   r23,
 		})
+
+		if !authenticatedOnly {
+			var r24 kline.Item
+			startTime, endTime := time.Now().AddDate(0, -1, 0), time.Now()
+			r24, err = e.GetHistoricCandles(p, assetTypes[i], startTime, endTime, kline.OneDay)
+			msg = ""
+			if err != nil {
+				msg = err.Error()
+				responseContainer.ErrorCount++
+			}
+			responseContainer.EndpointResponses = append(responseContainer.EndpointResponses, EndpointResponse{
+				Function:   "GetHistoricCandles",
+				Error:      msg,
+				Response:   r24,
+				SentParams: jsonifyInterface([]interface{}{p, assetTypes[i], startTime, endTime, kline.OneDay}),
+			})
+
+			var r25 kline.Item
+			r25, err = e.GetHistoricCandlesExtended(p, assetTypes[i], startTime, endTime, kline.OneDay)
```

Comment:
ZB throws a wrapper error for BTC-USDT but works via the CLI: zb GetSpotKline rawKlines is nil

Reply:
There are some issues with rate limiting on ZB I am fixing at the moment should be fixed next push

Reply:
I have enabled the rate limiter for exchange_wrapper_issues as it was disabled by default